res/js/tx.caretaker.NodeTree.js

```diff
@@ -354,7 +355,7 @@ tx.caretaker.NodeTree = Ext.extend(Ext.tree.TreePanel, {
 				"tx_caretaker[isInInstance]": (node.attributes.type == 'instance' ? 1 : 0),
 				"tx_caretaker[type]": 'test',
 				"tx_caretaker[storagePid]": this.storagePid,
-				"tx_caretaker[returnUrl]": encodeURIComponent(parent.list_frame.document.location)
```

Comment:
Why did you remove these encodeURIComponent()s?
I'm not sure if they are necessary, just wonder why you removed it